frontend-web/webclient/vite.config.ts

```diff
@@ -0,0 +1,70 @@
+import {defineConfig, ProxyOptions, UserConfigExport} from "vite";
+import reactRefresh from "@vitejs/plugin-react-refresh";
+import path from "path";
+import {DEV_SITE} from "./site.config.json";
+
+// https://vitejs.dev/config/
+
+type Mode = "development" | "local-dev" | "production" | "compose";
+
+function targetFromConfig(mode: Mode): string {
+    switch (mode) {
+        case "development":
+            return `https://${DEV_SITE}`;
+        case "compose":
+            return "http://backend:8080";
+        case "local-dev":
+        case "production":
+        default:
+            return "localhost:8080";
+    }
+}
```

Comment:
I want to make sure these targets are correct.

`development` is `npm run start`
`compose` is `npm run start:compose`
`local-dev` is `npm run start:local`
`production` is `npm run build`